File: JAVA/DPR.java
// Deklarasi kelas.
public class DPR 
{
    // Private atribut
    private String id;
    private String nama;
    private String bidang;
    private String partai;

    // Konstruktor.
    public DPR()
    {
        this.id = "";
        this.nama = "";
        this.bidang = "";
        this.partai = "";
    }

    public DPR(String nama, String bidang, String partai, String id)
    {
        this.id = id;
        this.nama = nama;
        this.bidang = bidang;
        this.partai = partai;
    }

    // Getter dan Setter.
    // Setter.
    public void setid(String id)
    {
        this.id = id;
    }
    public void setNama(String nama)
    {
        this.nama = nama;
    }
    public void setbidang(String bidang)
    {
        this.bidang = bidang;
    }
    public void setpartai(String partai)
    {
        this.partai = partai;
    }

    // Getter.
    public String getid()
    {
        return this.id;
    }
    public String getNama()
    {
        return this.nama;
    }
    public String getbidang()
    {
        return this.bidang;
    } 
    public String getpartai()
    {
        return this.partai;
    }
}

File: PHP/DPR.php
<?php 

// Inisialisasi kelas.
class DPR
{
    // Private attribute:
    private $id = '';
    private $nama = '';
    private $bidang = '';
    private $partai = '';
    private $gambar = '';

    // Konstruktor.
    public function __construct($id = '',$nama = '',  $bidang = '', $partai = '', $gambar = '')
    {
        $this->id = $id;
        $this->nama = $nama;
        $this->bidang = $bidang;
        $this->partai = $partai;
        $this->gambar = $gambar;
    }

    // Setter.
    // id.
    public function set_id($id)
    {
        $this->id = $id;
    }
    // Nama DPR.
    public function set_nama($nama)
    {
        $this->nama = $nama;
    }
    // Bidang.
    public function set_bidang($bidang)
    {
        $this->bidang = $bidang;
    }
    // partai.
    public function set_partai($partai)
    {
        $this->partai = $partai;
    }
    // Gambar.
    public function set_gambar($gambar)
    {
        $this->gambar = $gambar;
    }

    // Getter.
    // id.
    public function get_id()
    {
        return $this->id;
    }
    // Nama DPR.
    public function get_nama()
    {
        return $this->nama;
    }
    // Bidang
    public function get_bidang()
    {
        return $this->bidang;
    }
    // partai
    public function get_partai()
    {
        return $this->partai;
    }
    // Gambar
    public function get_gambar()
    {
        return $this->gambar;
    }

    // Destruktor.
    function __destruct()
    {
    }
}

?>